packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,140 @@
+# Chip Decisions <!-- omit in toc -->
+
+Chips embody a collection of selectable entities that empower users to engage in choices, sift through content, and invoke pertinent actions.
+
+This document outlines the API details of the `Chip` and `ChipGroup` components, encompassing their structural composition, functional attributes, and visual representation.
+
+<img width="70%" src="./chips-variants.png" alt="Different variants of the Chip component">
+
+- [Design](#design)
+- [`Chip` API](#chip-api)
+- [`ChipGroup` API](#chipgroup-api)
+- [Examples](#examples)
+  - [Basic/Uncontrolled Usage](#basicuncontrolled-usage)
+  - [Controlled Usage](#controlled-usage)
+- [Accessibility Considerations](#accessibility-considerations)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=52377%3A27686&mode=design&t=pFzCay0x4a6sTcoQ-1) to all variants of the `Chip` component.
+
+## `Chip` API
+
+> **Note**
+>
+> The `Chip` component is inherently tied to the `ChipGroup` and cannot be utilized outside its context.
+
+| Prop                    | Type      | Default     | Description                                                                                                                                        | Required |
+| ----------------------- | --------- | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| **children**            | `string`  | `undefined` | Renders the label text for the Chip                                                                                                                | ✅       |
```

Comment:
wouldn't be children because we might want to support the template strings `Bussiness ${api.busineessType}`. We shall do `StringChildrenType` from our `utils` which we have created for use in Radio/Checkbox/Text which is nothing but `React.ReactText | React.ReactText[];`

Reply:
Yes, in code I've used `StringChildrenType` only. Let me update it in the doc too.